### PR TITLE
Added VS2019 support

### PR DESCRIPTION
**Pull request by ncksol**
updated manifest file
upgraded project and solution for vs2019

### CODE REVIEW

File: SemanticColorizer/SemanticColorizer.csproj [hunk 1]
```diff
@@ -24,6 +25,7 @@
     <UseApplicationTrust>false</UseApplicationTrust>
     <BootstrapperEnabled>true</BootstrapperEnabled>
     <TargetFrameworkProfile />
+    <MinimumVisualStudioVersion>16.0</MinimumVisualStudioVersion>
```
Comment (clairernovotny) [inline SemanticColorizer/SemanticColorizer.csproj:28]: I don't think this is required and could interfere with building the solution on existing VS versions.

Comment (ncksol) [reply SemanticColorizer/SemanticColorizer.csproj:28]: I wasn't sure why VS2019 would put this in myself. But if you remove it and reopen the project in 2019 it "requires an update". I don't remember VS being so aggressive about versions.

Comment (clairernovotny) [reply SemanticColorizer/SemanticColorizer.csproj:28]: That's odd. I suppose you can still open/build it in VS 2017 for now?

Comment (ncksol) [reply SemanticColorizer/SemanticColorizer.csproj:28]: Yep. It opens and builds in VS2017 with no problems. Unfortunately I don't have any older versions to test this with.

**Comment by arBmind** [comment]
Thanks a lot!
I adopted the manifest changes, but kept the project files for now.

**Pull request closed without merging**